static/index.js

```diff
@@ -183,20 +183,26 @@ function makeId(length) {
 // create DOM Element for a device
 function showMachine(token, mId, device, clients) {
   var tr = document.createElement('tr');
+  if (mId === machineId) {
+    tr.classList.add('currentMachine');
+  }
   var td = document.createElement('td');
   var a = document.createElement('a');
   td.classList.add('machine');
-  a.textContent = '[x]';
-  a.onclick = function() {
-    unregisterMachine(mId)
-    .then(function(response) {
-      response.json()
-      .then(function(body) {
-        showMachines(token, body.machines, body.clients);
+  // unregister current machine only by using a button
```

Comment:
then we would need to have a different action for current one - forceUnregister one instead of reloading the page... which is certainly possible.